core/src/index/thirdparty/faiss/gpu/GpuIndexIVFFlat.cu

```diff
@@ -235,30 +235,33 @@ GpuIndexIVFFlat::searchImpl_(int n,
                              float* distances,
                              Index::idx_t* labels,
                              ConcurrentBitsetPtr bitset) const {
-  // Device is already set in GpuIndex::search
-  FAISS_ASSERT(index_);
-  FAISS_ASSERT(n > 0);
+                                   // Device is already set in GpuIndex::search
+    FAISS_ASSERT(index_);
+    FAISS_ASSERT(n > 0);
+    printf("Hello searchImpl_\n");
+    // Data is already resident on the GPU
+    printf("distance指针指向地址: %p", distances);
```

Comment:
Would you please remove the print log and change the Chinese comments to English?